test/e2e/framework/util.go

```diff
@@ -3641,6 +3641,40 @@ func RestartKubeProxy(host string) error {
 	return nil
 }
 
+func RestartKubelet(host string) error {
+	// TODO: Make it work for all providers.
+	if !ProviderIs("gce", "aws") {
+		return fmt.Errorf("unsupported provider: %s", TestContext.Provider)
+	}
+	var cmd string
+	if ProviderIs("gce") {
+		cmd = "sudo /etc/init.d/kubelet restart"
+	} else {
+		cmd = "sudo systemctl restart kubelet"
+	}
+	Logf("Restarting kubelet via ssh, running: %v", cmd)
+	result, err := SSH(cmd, host, TestContext.Provider)
+	if err != nil || result.Code != 0 {
+		LogSSHResult(result)
+		return fmt.Errorf("couldn't restart kubelet: %v", err)
+	}
+	return nil
+}
+
+func WaitForKubeletUp(host string) error {
+	cmd := "curl http://localhost:10248/healthz"
```

Comment:
no, because port is closed probably

10248 is defaulted here: https://github.com/kubernetes/kubernetes/blob/886e04f1fffbb04faf8a9f9ee141143b2684ae68/pkg/apis/componentconfig/v1alpha1/defaults.go#L261 (imo it should be added here, maybe I'll do it in another pr https://github.com/kubernetes/kubernetes/blob/c4eeb238ce19c514b7d3ba719f3311e36101eabe/pkg/master/ports/ports.go)